include/nccl-headers/nvidia/err.h

```diff
@@ -6,11 +6,13 @@
 #define NCCL_ERR_H_
 
 /* Error type for plugins */
-typedef enum { ncclSuccess                 =  0,
```

Comment:
We purposely don't change these headers from NCCL -- I think we should exclude everything in `include/nccl-headers` from formatting fixes.

Reply:
That's reasonable, I think it's as simple as adding a .clang-format file in the subdirectory.

As a future improvement I think we should look to restructure our include directories to make the visibility more clear:

1. internal headers for internal TUs under `src/include`
2. headers for consumers of this plugin (eg: ofi_nccl_api.h) at the top level `include/`
3. duplicated/upstream nccl headers under a third directory.